lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -102,11 +102,12 @@
                             })
                         },
                         {
-                            view: new  AccountSettingsFieldViews.DropdownFieldView({
+                            view: new AccountSettingsFieldViews.DropdownFieldView({
                                 model: userPreferencesModel,
                                 required: true,
                                 title: gettext('Time Zone'),
                                 valueAttribute: 'time_zone',
+                                hidden: !fieldsData.time_zone.enabled,
```

Comment:
Why the switch from `enabled` to `hidden` here? Instead of `hidden=!enabled` here, and `if(!hidden)` in the view, you could just have `enabled=enabled` and `if(enabled)`.